admin.php

```diff
@@ -104,14 +104,14 @@ function html() {
         echo '</table>';
 
         echo '<div class="pagenav">';
-        if($max < time()-(7*24*60*60)){
+        if ($max <= $_SERVER['REQUEST_TIME']){
```

Comment:
Thanks for your comment. I have now replaced `$_SEVER['REQUEST\TIME']` to `time()` call.